p2p/src/utils/requester.rs

```diff
@@ -78,29 +92,28 @@ impl<E: Clock + GClock, C: Scheme> Requester<E, C> {
         self.rate_limiter.shrink_to_fit();
     }
 
-    /// Once a participant is added to skip, it is never removed.
-    pub fn skip(&mut self, participant: PublicKey) {
+    /// Skip a participant for future requests.
+    ///
+    /// Participants added to this list will never be removed (even if dropped
+    /// during `reconcile`, in case they are re-added later).
+    pub fn block(&mut self, participant: PublicKey) {
```

Comment:
Once someone is blocked, we shouldn't reconsider interacting with them until restart.

This is needed to avoid spending resources verifying messages from people who typically send us invalid data (and to avoid waiting for them to respond).